Frontend/src/app/shared/components/create-project/create-project.component.scss

```diff
@@ -3,6 +3,7 @@
   padding: 20px 20px 20px 20px;
   background-color: white;
   width: min(500px, 95vw);
+  overflow: hidden;
```

Comment:
I added this overflow property because without it you can stretch the text area beyond the create project component and it did not look good for styling purposes, but If thats a quality we want this can be removed.